pg_metadata/install/sql/pgmetadata/20_TABLE_SEQUENCE_DEFAULT.sql

```diff
@@ -71,7 +71,7 @@ CREATE TABLE pgmetadata.dataset (
     spatial_extent text,
     creation_date timestamp without time zone DEFAULT now() NOT NULL,
     update_date timestamp without time zone DEFAULT now(),
-    geom public.geometry(Polygon,4326)
+    geom public.geometry
```

Comment:
I think we should keep use a geometry(Polygon,4326) here and use the `ST_Transform()` method to create the geometry from the table convex hull